### ISSUE TITLE
Mist connection node error

### ISSUE THREAD

**Issue by Birdscanfly** [ethereum/mist#1162]
#### System information
``````
![error](https://cloud.githubusercontent.com/assets/22262878/18610407/2f1c5d26-7d4e-11e6-8fdd-5fe4c5d429bc.png)

Version: `0.8.2`
OS & Version: Windows 10, 64 Bit
Node type: `eth/geth(default)`
``````
I totally can't access Mist at all. Pretty much stuck at the front screen.

Also, Mist has been working ok before. It just suddenly threw up this error only today.

Thanks.

**Comment by luclu** [comment]
Make sure you did extract the files before running. 

Also try the following:
1. Make sure your AV didn't delete any files.
2. Re-download a fresh copy and extract it.
3. Rename (backup and delete) the `%APPDATA%\Ethereum\chaindata` folder and retry.
4. Rename the `%APPDATA%\Mist` folder.

**Comment by Birdscanfly** [comment]
Hello, thanks for the reply. I did what you said and it did not work, sadly. It now throws up this error instead

![error 2](https://cloud.githubusercontent.com/assets/22262878/18614901/8c16125c-7dca-11e6-838a-6530f1ba1ff0.png)

The files were run after extraction, of course

**Comment by luclu** [comment]
@Birdscanfly, we just published [release `0.8.6`](https://github.com/ethereum/mist/releases/tag/v0.8.6) which will download **geth** `1.4.18` on-the-fly. 
*Please* update to easily roll with the upcoming [Hard Fork](https://blog.ethereum.org/2016/10/13/announcement-imminent-hard-fork-eip150-gas-cost-changes/) on Tuesday.
